drivers/common/opensensors.h

```diff
@@ -0,0 +1,30 @@
+#ifndef __OPENSENSORS_H__
+#define __OPENSENSORS_H__
+
+#define MAXSENSORS           10
```

Comment:
Earlier sensors.h was included into bsp/boards, now should be created for each board (e.g., OpenMote-CC2538). sensors.h is now in charge of defining MAXSENSORS